amplfi/law/amplfi/law/base.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from amplfi.law.paths import paths
+
+
+class AmplfiDataTaskMixin:
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
What is the super class here?

Reply:
This is meant to be a "Mixin" for `law` `SandboxTasks`, specifically for the use case of appending environment variables. You can see it's use e.g. [here](https://github.com/ML4GW/amplfi/blob/19cdf5ba5646f7724f2ba0976ea1596e90f4a318/amplfi/law/amplfi/law/tasks.py#L24)

For reasons that are hard to explain over github, it didn't make sense to subclass the `mldatafind` tasks and override the `sandbox_env` due to additional boiler plate that would induce

Reply:
OK. But then add comments to which method in the law/mldatafind source code is being invoked when you call `super()`.

Reply:
done